### DISCUSSION TITLE
求助，目前这些角色哪些更值得培养？

### DISCUSSION THREAD

**Discussion by Archon0203** [share121/inter-knot#111] [其他]
![1723198674871](https://github.com/user-attachments/assets/4b1b9064-383d-487f-a2dc-c7c2492a4447)
我只有这几个代理人，目前游玩体验还算可以，但是我担心后期压力太大，所以优先培养谁好一点呢？

**Comment by DtarDomeExploit** [comment]
不要有什么压力，你可以当成单机游戏玩，哪怕通不过深渊，也可以游戏不会爆炸，米家游戏最好的游玩心态是把他当一个42天一更的可玩番剧，如果真的要推荐几个后期比较有用的话，如果你没有手的话，可以把大熊本练一下，因为大后期零号空洞没有手，就得靠大熊本这个轮椅了，啊，大熊本我们手残党最可靠的伙伴

**Comment by Z-Century** [comment]
没好击破的话安比一定要养

**Comment by Gerwsr** [comment]
太可恶了，我说比利今天怎么找不到了，原来跑你这来了，快把比利还给我！(•́へ•́ ╬)

**Comment by Trashcanraccoon** [comment]
可莉！！性价比超高，强力输出，手感也很好，我现在最喜欢的就是可莉了~

**Reply by Trashcanraccoon** [reply]
不对！可琳！

**Comment by TraitYoung** [comment]
可琳不错，手感好伤害也不错，最主要的是可爱捏

**Comment by chenger2024** [comment]
安东目前还行，可以放心培养

**Comment by Elysia-Ego-093** [comment]
首推安比，超高性价比击破手，手感十分舒服。
